django/contrib/admin/static/admin/js/SelectBox.js

```diff
@@ -18,20 +18,19 @@
             // Repopulate HTML select box from cache
             var box = document.getElementById(id);
             var node;
-            $(box).empty(); // clear all options
-            var new_options = box.outerHTML.slice(0, -9); // grab just the opening tag
+            while (box.options.length) {
+                box.options.remove(0);
+            }
```

Comment:
Question: rather than this loop, can we not just `box.options.innerHTML = ''` here?

Reply:
@carltongibson yes, that would actually be better since my method only removes the `option` not the `optgroup` children. Thanks!